src/main/webapp/user_profile_script.js

```diff
@@ -52,10 +104,105 @@ function closeModal() {
     modal.style.display = "none";
 }
 
+function closeEditModal() {
+    var modal = document.getElementById("editTaskModal");
+    modal.style.display = "none";
+}
+
 // If the user clicks outside of the modal, closes the modal directly
 window.onclick = function(event) {
     var modal = document.getElementById("createTaskModal");
     if (event.target == modal) {
         modal.style.display = "none";
     }
+    var editModal = document.getElementById("editTaskModal");
+    if (event.target == editModal) {
+        editModal.style.display = "none";
+    }
+}
+
+async function displayNeedHelpTasks() {
+    const queryURL = "mytasks?keyword=Owner";
+    const request = new Request(queryURL, {method: "GET"});
+    const response = await fetch(request);
+    const taskResponse = await response.json();
+    const needHelpTable = document.getElementById("need-help");
+    needHelpTable.innerHTML = "";
+    var headerRow = document.createElement("tr");
+    var headers = ["Task Overview", "Helper", "Status", "Edit", "Delete"];
+    for (var index = 0; index < headers.length; index++) {
+        var th = document.createElement("th");
+        th.appendChild(document.createTextNode(headers[index]));
+        headerRow.appendChild(th);
+    }
```

Comment:
I'd prefer this static HTML to live in the HTML file; you can use JS to hide/show it as needed.

Reply:
The reason why I add a header with js code every time is that I haven't found a proper way to remove the other rows without removing the first row. The only way I knew that can remove multiple rows in one step is to set the innerHtml to empty string. But in this way, the static header will also be removed. So I add the header rows dynamically here. I will try to find a way to use static HTML in a new PR.

Reply:
I just found a way to replace the innerhtml of a table to the inner html of the first row of that table. Now I am able to use a static row header HTML instead of adding the heading row each time I display the table. I will make the change in the new commit.